### ISSUE TITLE
[SF][MultiInput][A11y]:ARIA listbox missing required child roles group or option

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents#13707]
### Bug Description

The ui5-tokenizer component renders a <div> with role="listbox" but contains no children with role="option" or        role="group", violating the ARIA required children rule and causing accessibility violations.                         
`<div class="ui5-tokenizer--list" role="listbox" aria-label="Tokenizer"></div>  ` 
                                      
A listbox role requires at least one child with role="option" or role="group" per the ARIA specification. When the tokenizer is empty, no such children are present, triggering the following axe-core violation:                        
                                                                                                                        
  ▎ Certain ARIA roles must contain particular children — expecting ARIA children role to be added: group, option  

<img width="1878" height="204" alt="Image" src="https://github.com/user-attachments/assets/26fa9bac-867e-4eeb-aec5-2a492d6b6287" />

### Affected Component

MultiInput

### Expected Behaviour

 When the tokenizer contains no tokens, the container element should either:                                           
  - Not use role="listbox" (e.g., remove the role or use a different semantic element), or
  - Ensure at least a placeholder child with an appropriate role is present so that the ARIA required children rule is  
  satisfied. 

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.23.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @tandan0813 ,

The **Tokenizer** has slotted `ui5-token` components. Their root/wrapper element `has role="option"` as shown on the screenshot.

<img width="665" height="150" alt="Image" src="https://github.com/user-attachments/assets/480c8b8c-9873-4ed0-a6fd-84e95288a602" />

I can't reproduce the mentioned Axe error when the MultiInput is empty on this link:
https://ui5.github.io/webcomponents/components/MultiInput/ (2.23.1)

<img width="1707" height="1270" alt="Image" src="https://github.com/user-attachments/assets/f4f48f02-65fe-43d3-84f0-673661e6f644" />

Could you please re-check your scenario and give us isolated sample if it's still reproducible ?

Regards,
Plamen Ivanov
